server/src/main/java/org/elasticsearch/search/aggregations/AggregatorFactory.java

```diff
@@ -226,28 +226,20 @@ public void doValidate() {
 
     protected abstract Aggregator createInternal(SearchContext searchContext,
                                                     Aggregator parent,
-                                                    boolean collectsFromSingleBucket,
+                                                    CardinalityUpperBound cardinality,
                                                     Map<String, Object> metadata) throws IOException;
 
     /**
-     * Creates the aggregator
+     * Creates the aggregator.
      *
-     *
-     * @param searchContext
-     *            The search context
```

Comment:
+1 for dropping this comment.  3 lines to say nothing the type declaration didn't.